conda_forge_tick/git_utils.xsh

```diff
@@ -135,7 +135,8 @@ def get_repo(attrs, branch, feedstock=None, protocol='ssh',
         # git pull @(upstream) master -s recursive -X theirs --no-edit
         # always run upstream master
         git fetch @(upstream)
-        git reset --hard @(upstream)/master
+        git remote add ups @(upstream)
```

Comment:
Please call this "upstream" for clarity.  I am all for abbreviations, on personal machines, but no need for shortcuts here

Reply:
I didn't want to confuse it with `upstream` which is a python variable (it might be a bit jarring for someone new to xonsh)

Reply:
boh